server/entity/MediaRequest.ts

```diff
@@ -111,6 +111,7 @@ export class MediaRequest {
           image: `https://image.tmdb.org/t/p/w600_and_h900_bestv2${movie.poster_path}`,
           notifyUser: this.requestedBy,
           media,
+          requestId: this.id,
```

Comment:
Would about just passing the request object as we do with media?

Reply:
Oh yeah we should totally do that!